### PR TITLE
add cool features

### PR DESCRIPTION
**Pull request by Wzixiao**

### CODE REVIEW

**Comment by Wzixiao** [comment]
添加了一些很酷的功能

- 请求时会有动画提示
- 在请求时可以去别的单元格写代码，只要请求完毕，按下Tab会回到请求的单元格并把请求的代码写出来

File: examples/sklearn_digits.ipynb [hunk 1]
```diff
@@ -5,64 +5,47 @@
    "execution_count": 1,
```
Comment (liyongsea) [inline examples/sklearn_digits.ipynb:5]: Do not add notebook modif in the PR

File: examples/.ipynb_checkpoints/sklearn_digits-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (liyongsea) [inline examples/.ipynb_checkpoints/sklearn_digits-checkpoint.ipynb:1]: do not add checkpoint in commit

File: content.js [hunk 3]
```diff
@@ -44,39 +58,48 @@ Please provide the missing code to complete the task. (do not include code that
...
-        });
+
+        // 开始动画
+        const [animationInterval, animationElement] = startWaitingAnimation(activeCell)
+
+        isRequtest = true
+        const suggestion = await getCodeCompletion(code)
+        if(suggestion){
+          // Use a regular expression to match the content between triple backticks
+          const codeBlockRegex = /```([\s\S]*?)```/g;
```
Comment (liyongsea) [inline content.js:90]: put lines 90-93 into getCodeCompletion such that you can switch openai to another codecompletion tools

**Review by liyongsea — APPROVED** [review]

**Pull request merged**